packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -538,12 +538,25 @@ class TextSelectionOverlay {
   }
 
   void _handleSelectionHandleChanged(TextSelection newSelection, {required bool isEnd}) {
-    final TextPosition textPosition = isEnd ? newSelection.extent : newSelection.base;
+    final TextPosition bringTextPosition;
```

Comment:
Nit: Maybe `positionToView` or something like that is more clear?

Reply:
agree